### PR TITLE
Dev/monthly report tag

### PR DESCRIPTION
**Pull request by bakunyo**
https://github.com/hr-dash/hr-dash/pull/110 に依存
- タグ入力周りの整理
- 月報閲覧画面のMarkdown, Tag対応など

### CODE REVIEW

File: app/views/layouts/_monthly_report_tags.html.erb [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+<script>
+$(function(){
+  var refresh_tags_input = function(self) {
+    var input = $('#monthly_report_tags_input');
+    (typeof(input) == undefined) ? null : input.val(self.getTags());
+  }
+
+  var tagger = $('#monthly_report_tags').tags({
+      tagData: <%= raw (tags.present? ? tags.map(&:name) : []) %>,
+      suggestions: <%= raw Tag.all.pluck(:name) %>,
```
Comment (freezer530000) [inline app/views/layouts/_monthly_report_tags.html.erb]: **[brakeman]**
- Unescaped model attribute [:ref](http://brakemanscanner.org/docs/warning_types/cross_site_scripting)

File: app/controllers/monthly_reports_controller.rb [hunk 2]
```diff
@@ -1,8 +1,59 @@
...
+      render :new
+    end
+  end
+
+  private
+
+  def assign_relational_params(report)
+    report.status = params[:wip] ? :wip : :shipped
+    report.monthly_working_processes = working_processes(report)
+    report.tags   = monthly_report_tags
```
Comment (freezer530000) [inline app/controllers/monthly_reports_controller.rb]: **[rubocop]**
- Unnecessary spacing detected.

**Comment by Nobuhiko-Kudo** [comment]
:ok:

**Comment by ta-chibana** [comment]
:elephant:

**Pull request merged**